portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -825,6 +825,39 @@ definition {
 			locator1 = "TextInput#ANY");
 	}
 
+	@description = "LPS-177999. Verify that URL, description, type and source code url fields are present."
+	@priority = 5
+	test ThemeSvgTypeCanDisplayProperFields {
+		property custom.properties = "feature.flag.LPS-166479=true";
+		property portal.acceptance = "true";
+
+		task ("Given Client Extension admin page") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+		}
+
+		task ("When Create a Theme svg extensions") {
+			ClientExtensionGeneral.addType(type = "Add Theme SVG Spritemap");
+		}
+
+		task ("Then URL, description, type and source code url fields are present") {
+			ClientExtensionGeneral.assertContentField(
+				entryField = "Name",
+				entryInformation = "");
+
+			AssertTextPresent(
+				locator1 = "ClientExtension#THEME_SVG_DESCRIPTION_FIELD",
+				value1 = "Description");
```

Comment:
```suggestion
				ClientExtensionGeneral.assertContentField(
				entryField = "description",
				entryInformation = "");
```

You dont need create a new path, try to use this why, I think it will work :)